x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/Job.java

```diff
@@ -244,6 +244,17 @@ public Job(StreamInput in) throws IOException {
         deleted = in.readBoolean();
     }
 
+    /**
+     * Get the persisted job document name from the Job Id.
+     *
+     * @param jobId The job id
+     * @return The id of document the job is persisted in
+     */
+    public static String documentId(String jobId) {
+        return "job-" + jobId;
```

Comment:
since we're back at having the jobs indexed, we should double check the length limit on the ID is correct